### DISCUSSION TITLE
Fix last active when using biometrics/pin login in browser

### DISCUSSION THREAD

**Discussion by phyzical** [bitwarden/clients#5046] [Password Manager]
### Code Contribution Proposal

Not sure if the process is now to create one of these when the community submits a pr but will anyway,

https://github.com/bitwarden/clients/pull/4835

Basically when performing any login that does not interact with the popup window the last active is not updated and thus you are logged out on the next interval tick.

this pr just adds a last active update onblur which occurs when the popup window closes which does occur in these instances.

**Comment by dwbit** [comment]
Thanks! The team will review the PR and get back to you shortly.

**Comment by djsmith85** [comment]
Closing the discussion, as this was released with `2023.5`
Thanks again @phyzical for your contributions ❤️
